lms/static/sass/discussion/utilities/_shame.scss

```diff
@@ -50,6 +54,11 @@
   @include padding-left(0);
 }
 
+.forum-nav-browse-menu-item {
```

Comment:
why `shame` for this? Seems non-shameful to me, in which case it should go in `lms/static/sass/discussion/elements/_navigation.scss`